cis/packaging/postinst

```diff
@@ -1,12 +1,38 @@
 #!/bin/bash
 CONFIGURATION_PATH=/var/rudder/packages/rudder-plugin-cis
+FOLDERS="ncf/30_generic_methods/cis techniques/CIS_plugin"
```

Comment:
Why put a space in the name?

Reply:
no space it is two separated folders, used in for loop